Sources/Overload/OvCore/include/OvCore/ECS/Actor.h

```diff
@@ -100,6 +100,12 @@ namespace OvCore::ECS
 		*/
 		void DetachFromParent();
 
+		/**
+		* Returns true if this actor transform is descendant of the actor
+		* @param p_actor
+		*/
+		bool IsDescendantOf(const Actor* p_actor) const;
```

Comment:
I believe this should be exposed to Lua through the `LuaActorBinder`.
The wiki should also be updated to include it.

Reply:
Comment addressed in the latest commit, I will update the wiki when the MR will be approved.